### DISCUSSION TITLE
Create Example Config fails with

### DISCUSSION THREAD

**Discussion by PeterQuinn925** [bellrichm/WeeWX-MQTTSubscribe#209] [General]
New install on a fresh VM running Debian. Installed Weewx V5.0.2 using the package install. I don't quite understand the V5 changes having to do with the weewx user vs running root, but I got it working. I'm testing out MQTT Subscribe in preparation to switching over my real system to use it.

Following the instructions for Create Example Config, using the command 
python3 /etc/weewx/bin/user/MQTTSubscribe.py --create-example mqttsubscribe.template.conf

I get:

  File "/etc/weewx/bin/user/MQTTSubscribe.py", line 392, in <module>

    import weeutil

ModuleNotFoundError: No module named 'weeutil'

weeutil exists in /usr/share/weewx/ so I'm assuming there is a path issue. I expect I can add it to a path and get it to work.  I'll report back when I have a chance to look at it again.

**Comment by bellrichm** [comment]
The first question is, what release of MQTTSubscribe? That only works for v3+, which is in beta.

If you installed from, https://github.com/bellrichm/WeeWX-MQTTSubscribe/archive/refs/tags/latest.zip, you probably got a bad release. If you really want v3, try https://github.com/bellrichm/WeeWX-MQTTSubscribe/archive/refs/tags/v3.0.0-rc08.zip.

Assuming you want v3, in the same terminal try `echo $BIN_ROOT`. For kicks, check all the environment variables, $WEEWX_REPO , $WEEWX_ROOT,  $USER_ROOT, $CONFIG_FILE, $WEECTL.

If you want v2, follow these instructions, https://github.com/bellrichm/WeeWX-MQTTSubscribe/wiki/Installing-and-updating-version-2.x-and-earlier

**Comment by PeterQuinn925** [comment]
I'm thinking about installing it on my real system once I understand how it works and get some experience with it. I used the instructions and got "latest.zip". I'll try again with the recommended zip.

I don't see the point in using an older version, especially if you're at the release candidate stage.

Thanks!
